include/aos/sm/resourcemanager.hpp

```diff
@@ -22,7 +22,7 @@ namespace resourcemanager {
  * Unit config contains single node unit details.
  */
 struct UnitConfig {
-    NodeUnitConfig            mNodeUnitConfig;
+    NodeConfig                mNodeConfig;
```

Comment:
This is also node config but with version. And as they are in different namespaces, we can name this structure as `NodeConfig` also?

Reply:
JSON provider uses this struct in interface methods.
This sturct is dumped/parsed into/from JSON.